src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -327,6 +342,54 @@ public Command getTrajectoryFollowingCommand(String trajectoryName) {
                 },
                 this).andThen(() -> stop());
 
-        return command;
-    }
+		return command;
+	}
+
+	//get best pose to shoot based on Apriltag pose, the pose you want to go to
+	public Pose2d getTagRobotPose(boolean shiftLeft, boolean shiftRight){
+		Pose2d tagPose = m_vision.getTagPose(m_vision.getBestTagID()).get(); //get AprilTag pose of best ID tag
```

Comment:
I believe what @prensing meant was that, the vision should only return the best pose in the image currently and the driveTrain should never query for anything with the Apriltags ID, etc. 
i.e. the driveTrain shouldn't know anything about what tag ID to look for at all